persistence-modules/hibernate-queries/pom.xml

```diff
@@ -49,9 +81,34 @@
             <artifactId>jmh-generator-annprocess</artifactId>
             <version>${jmh-generator.version}</version>
         </dependency>
-    </dependencies>
+        
+        <dependency>
+			 <groupId>com.sun.xml.bind</groupId>
+			 <artifactId>jaxb-core</artifactId>
+			 <version>${com.sun.xml.version}</version>
+		</dependency>
+		<dependency>
+			 <groupId>javax.xml.bind</groupId>
+			 <artifactId>jaxb-api</artifactId>
+			 <version>${javax.xml.bind.version}</version>
+		</dependency>
+		<dependency>
+			 <groupId>com.sun.xml.bind</groupId>
+			 <artifactId>jaxb-impl</artifactId>
+			 <version>${com.sun.xml.version}</version>
+		</dependency>
+    </dependencies> 
 
     <properties>
+   		<!-- Spring -->
+        <org.springframework.version>5.0.2.RELEASE</org.springframework.version>
+        <org.springframework.data.version>1.10.6.RELEASE</org.springframework.data.version>
+        <!-- persistence -->
+        <hibernate.version>5.2.10.Final</hibernate.version>
```

Comment:
By providing the hibernate.version explicitly here you overwrite the one provided in the parent pom.xml and actually downgrade it. Is it intended?

Reply:
Hibernate.version has been removed. The comment has been addressed.